Package.swift

```diff
@@ -39,4 +39,3 @@ let package = Package(
     ],
     swiftLanguageVersions: [.v5]
 )
-let version = Version(0, 3, 0)
```

Comment:
This line isn't needed. SPM works with the current tag (there is no mention of this field in the current [Package documentation](https://developer.apple.com/documentation/swift_packages/package)). Thank you @GeorgeLyon for teaching me this!